app/services/process_refund_service.rb

```diff
@@ -17,17 +19,26 @@ def run(finance_details:, payment:, user:)
 
   private
 
-  attr_reader :payment, :user
+  attr_reader :payment, :user, :finance_details
 
   def card_refund_result
-    @_card_refund_result ||= ::Worldpay::RefundService.run(payment: payment)
+    @_card_refund_result ||= ::Worldpay::RefundService.run(payment: payment, amount: amount_to_refund)
+  end
+
+  def amount_to_refund
+    # We can never refund unless there have been an overpayment.
+    return 0 unless finance_details.balance.negative?
+
+    overpayment = (-1 * finance_details.balance)
```

Comment:
Purely because I'm slow on the uptake could we add a comment that just explains what this line is doing?

```suggestion
    # A quick maths trick to convert a negative value to a postive one
    overpayment = (-1 * finance_details.balance)
```